src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs/launch_analysis.cljs

```diff
@@ -143,7 +143,7 @@
                                       (select-keys props [:config-id :workspace-id :root-entity-type :on-success])
                                       {:dismiss-self #(swap! state dissoc :display-modal?)})])}])
       [comps/Button {:text "Launch Analysis..."
-                     :disabled? (when (:disabled? props) "The workspace is locked")
+                     :disabled? (when (:disabled? props) (:disabled? props))
```

Comment:
`(when (:disabled? props) (:disabled? props))` is logically equivalent to just `(:disabled? props)`